src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.CommandType;
+import duke.command.DeadlineCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.EventCommand;
+import duke.command.FindCommand;
+import duke.command.InvalidCommand;
+import duke.command.ListCommand;
+import duke.command.TodoCommand;
+
+/**
+ * Parses and processes user input.
+ */
+public class Parser {
+
+    /**
+     * Returns a CommandType object which identifies the type of command was passed by the user.
+     *
+     * @param command a string which when valid, should correspond to one of the CommandType objects
+     * @return CommandType object
+     * @throws DukeException if the command string does not correspond to any of the valid CommandType objects
```

Comment:
Should these descriptions start with a capital letter and end with a punctuation? I noticed this in a few other javadoc comments too.